### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by denis-yuen**
Creating a pull request to comment on specific lines of documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ This service is aimed at developers familiar with Dockstore (and have at least g
 It also has the following limitations
 
 1. The service lacks a GUI and is purely a tool provided for developers doing conversion
-2. It is not possible to create build triggers in Quay.io programmatically at this time. So new refresh code in Dockstore 1.2 is required.
+2. It is not possible to create build triggers in Quay.io programmatically at this time. So new refresh code in Dockstore 1.2 was added to detect metadata added to quay.io repos.
```
Comment (denis-yuen) [inline README.md:75]: I noticed that the metadata is not described. 
This should be described after `#### Result:`

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ This service is aimed at developers familiar with Dockstore (and have at least g
...
 3. The service and client only handles local files.  It currently does not handle file provisioning.
 4. The Dockerfile uploaded will be renamed to "Dockerfile" on GitHub.  This is due to a Quay.io limitation, it will only build the "Dockerfile" in a GitHub archive.  
 5. The CWL descriptor uploaded will be renamed to "Dockstore.cwl" on GitHub.  This is due to the Write-API-Client Publish command and Dockstore limitation.  The input json for the Publish command does not contain any information regarding the descriptor name.  The Dockstore automatic refresh code will only try to import "Dockstore.cwl".
```
Comment (denis-yuen) [inline README.md:78]: Need counter or some message for 5, 4, etc. minutes during build

File: README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ This service is aimed at developers familiar with Dockstore (and have at least g
 It also has the following limitations
```
Comment (denis-yuen) [inline README.md:72]: Travis-CI badge oddly links to a SVG picture of a badge rather than the build on Travis-CI

**Review by garyluu — APPROVED** [review]

**Comment by denis-yuen** [comment]
FYI, I don't believe that the two first comments have been addressed, at least not in this branch

**Comment by garyluu** [comment]
Ready for re-review

**Comment by denis-yuen** [comment]
Looks good

**Pull request merged**